forta-bots/Bridge-Invariant/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "maker-bridge-invariant",
+  "version": "0.0.1",
+  "description": "Bot to monitor the Maker Bridge Invariant across the L2 networks",
+  "chainIds": [
+    1
+  ],
+  "scripts": {
+    "build": "tsc",
+    "start": "npm run start:dev",
```

Comment:
can we use yarn (both for lockfiles and for running scripts)? this is what we use in all other projects in this repo.

Reply:
I guess it is possible.

Reply:
changed from npm to yarn :heavy_check_mark: